src/db/Storage.java

```diff
@@ -0,0 +1,11 @@
+package db;
+
+import model.Bet;
+
+import java.util.ArrayList;
+import java.util.List;
+
+//ЭТО НАША БД
+public class Storage {
+    public static final List<Bet> bets = new ArrayList<>();
```

Comment:
```suggestion
    public static final List<Bet> BETS = new ArrayList<>();
```

Reply:
Unfortunately, most classes wrote German, for the interface, he himself said to write it separately. 
Everything else is changed according to the comments.